modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-s3-compatible-backup-storage-providers_{context}"]
+= S3 - Compatible backup storage providers
+
+
+OADP supports a variety of object storage providers for different backup and snapshot operations.
+
+== S3 drivers for Backup Storage Locations
```

Comment:
I think this probably should be AWS S3 drivers

Reply:
@sseago Good question. When we are talking about a resource type, it's `BackupStorageLocation`. But here, we're talking about them in general. See callout 9 in step 3 here: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/backup_and_restore/index#oadp-installing-dpa_installing-oadp-azure.  The usage in the heading is the general case.

Reply:
@RichardHoch If the usage is not referring to the in-cluster resource (i.e. you're referring to an s3 bucket in general), then it's probably best to use a more generic term. "backup storage location" is so close to `BackupStorageLocation` that it appears that they refer to the same thing. Maybe something like "Specify a bucket to store backups in" or at least something that doesn't match the wording of the resource type exactly.